prjxray/overlay.py

```diff
@@ -0,0 +1,60 @@
+class Overlay(object):
+    """ Object that represents an overlay.
+
+    Can be used to iterate over tiles and sites not inside a partition region.
+
+    """
+
+    def __init__(self, db, region_dict):
```

Comment:
Why not just take the grid?

Reply:
Which grid are you talking about?  I'm trying to match the original roi input schema which was x_max, x_min, y_min, y_max.  Just abstracting that to take multiple regions.

Reply:
You take the `db` argument, but immediately grab the grid, and discard the db.  Just take the grid argument.